src/util/constants/ArticleStatus.ts

```diff
@@ -7,18 +7,14 @@
  */
 
 export const ARCHIVED = 'ARCHIVED';
```

Comment:
I og med at statuser for søk hentes fra state-machine så trengs vel egentlig ikkje denne fila lenger? Heller ikkje ConceptStatus?

Reply:
Statusene brukes fortsatt i koden til diverse sjekker